platform/kube/conversion.go

```diff
@@ -43,6 +43,10 @@ const (
 
 	// IstioURIPrefix is the URI prefix in the Istio service account scheme
 	IstioURIPrefix = "spiffe"
+
+	// PortAuthenticationAnnotationKeyPrefix is the annotation key prefix that used to define
+	// authentication policy.
+	PortAuthenticationAnnotationKeyPrefix = "tls.istio.io"
```

Comment:
How about authpolicy.istio.io  - TLS is just one protocol for authentication. Maybe we'll use JWT, or QUIC (which is UDP and not real TLS), or ipsec transport - the setting is to specify the policy or config for encryption, not to customize tls.